### DISCUSSION TITLE
keyball39 パッドの剥離について

### DISCUSSION THREAD

**Discussion by 7shp2999** [Yowkees/keyball#384] [Q&A]
ctrlキー（18番の上のキー）のパッド2つが剥離してしまいました。少しパターンが持っていかれてます。
修復可能でしょうか？
ご教示頂けますと幸いです。よろしくお願いします。

![image](https://github.com/Yowkees/keyball/assets/49140767/9e70d5bc-c2a4-4fc6-85b9-b0c077093a44)

**Comment by Yowkees** [comment]
ご連絡ありがとうございます。

下画像のオレンジ点どうし、青点どうしをエナメル線やゴム被覆導線で接続していただければ復活できると思います。（パターン剥がれも大丈夫です）
なお、キーソケットの保持力がまったくなくなりますので、キーソケットをボンドや両面テープ等で固定してから作業していただくと良いと思います。

以上、お手数ですがご対応お願いいたします。

![BK39_ctrlfix](https://github.com/Yowkees/keyball/assets/84209772/6d37179c-1cb0-4f44-ad02-5cbcfe852891)

**Reply by 7shp2999** [reply]
丁寧にご回答いただきまして、大変ありがとうございました。早速試してみたいと思います。
